examples/USER/phonon/dynamical_matrix_command/Silicon/ff-silicon.lmp

```diff
@@ -16,4 +16,4 @@ mass ${Si}     28.08550
 ###########################
 
 pair_style      tersoff
-pair_coeff      * *  Si.opt.tersoff Si(D)
+pair_coeff      * *  ../../../../../potentials/SiCGe.tersoff Si(D)
```

Comment:
rather than using relative path in the input file, i would suggest to use a symbolic link. that is what is done in many examples